app/components/chat/chat-input.tsx

```diff
@@ -155,6 +155,7 @@ export default function ChatInput(props: ChatInputProps) {
     if (isURL(input)) {
       setTemporaryURLInput(input);
     }
+    setUserInput("");
```

Comment:
if I add after the `callLLM` the text input only clears after the request, and it gives a laggy feeling